misc/openapi-ballerina/modules/openapi-validator/build.gradle

```diff
@@ -25,8 +27,54 @@ configurations.all {
 dependencies {
     implementation 'io.swagger.parser.v3:swagger-parser'
 
-    implementation project(':ballerina-tool')
+    // Standard libs
     implementation project(':ballerina-lang')
+    implementation project(':language-server:language-server-compiler')
+
+    implementation project(':ballerina-runtime')
```

Comment:
Can we check why do we need all this std lib dependencies ?

Reply:
previously used this for creating the tests run, we can remove these. I will remove unused dependencies.